src/components/button/genericButton.js

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
you can improve it by adding on show property like when to show when not to show